extension-requests/local-utils.js

```diff
@@ -66,6 +66,25 @@ async function getTaskDetails(taskId) {
   return await res.json();
 }
 
+async function getUserDetails(username) {
+  if (!username) return;
+  const url = `${API_BASE_URL}/users?search=${username}&size=1`;
+  const res = await fetch(url, {
+    credentials: 'include',
+    method: 'GET',
+    headers: {
+      'Content-type': 'application/json',
+    },
+  });
```

Comment:
We have makeApiCall util function for fetching the data can you use that function?

Reply:
All the previous api calls in this file are using fetch. I have noted this down as refactor task in this issue: #430